app.js

```diff
@@ -163,53 +167,41 @@ function convertToSheetName(artifactName) {
     $.ajax({
       method: "GET",
       crossDomain: true,
-      url: userInfo.spiraUrl + 'services/v5_0/RestService.svc/projects' + userInfo.auth,
+      url: userInfo.spiraUrl + 'services/v5_0/RestService.svc/projects' + atob(userInfo.auth),
       success: function (data, textStatus, response) {
+        //if call is successful, fill the projects drop down with the user's projects and
+        //transition to the main screen
         for (let i = 0; i < data.length; i++) {
           $('<option value="' + data[i].ProjectId + '">' + data[i].Name + '</option>').appendTo('#projects');
         }
+        $('#current-user').html("Logged in as: " + userInfo.username);
         $('#logInScreen').addClass("hidden");
         $('#mainScreen').removeClass("hidden");
+        $('#log-out').removeClass("hidden");
         enableButtons();
       },
       error: function () {
         $('#username').addClass("error");
         $('#apikey').addClass("error");
-        $('#error-message').text(" Invalid Username or API key").addClass("ms-Icon ms-Icon--Error");
+        $('url').addClass("error");
+        $('#error-message').html("Invalid Login Info");
         enableButtons();
       }
     });
   }
 
-  function showHelp() {
-    $('#chevron-icon').toggleClass("ms-Icon--ChevronRight");
-    $('#chevron-icon').toggleClass("ms-Icon--ChevronDown");
-    $('#help-text').toggleClass("hidden");
-  }
-
-  //for testing calls and functions with a temporary "test" button on index.html
-  function testing() {
-    return Excel.run(function(context){
-        let sheetName = convertToSheetName("requirements");
-        let sheet = context.workbook.worksheets.getItem(sheetName);
-        let testCell = sheet.getCell(3, 0);
-        testCell.load();
-        return context.sync()
-        .then(function(){
-          let val = testCell.values[0][0];
-          val = multilistConvert(val);
-        });
-    });
-  } //end of testing
-
-  // The initialize function must be run each time a new page is loaded
+  // The initialize function must be run each time a new page is loaded. Currently there is only one page.
   Office.initialize = function (reason) {
```

Comment:
Again there's a lot going on here, definitely needs some comments!

Also ` $( "selector1, selector2" ).etc etc` could clean up the code a bit.